config-communicate.json

```diff
@@ -40,6 +40,9 @@
     "default_theme": "light",
     "feature_custom_themes": true,
     "auto_redirect_from_welcome_screen_to_login": true,
+    "logo_authheader_act": {      
```

Comment:
Please remove all references to "act" e.g. logo_authheader_act => logo_authheader

Reply:
done replaced act with secondary